PowerPointLabs/PowerPointLabs/ActionFramework/PasteLab/PasteToFillSlide/PasteToFillSlideActionHandler.cs

```diff
@@ -14,7 +14,7 @@ class PasteToFillSlideActionHandler : PasteLabActionHandler
         protected override ShapeRange ExecutePasteAction(string ribbonId, PowerPointPresentation presentation, PowerPointSlide slide,
                                                         ShapeRange selectedShapes, ShapeRange selectedChildShapes)
         {
-            ShapeRange pastingShapes = ClipboardUtil.PasteShapesFromClipboard(slide);
+            ShapeRange pastingShapes = ClipboardUtil.PasteShapesFromClipboard(presentation, slide);
             if (pastingShapes == null)
             {
                 return null;
```

Comment:
Does hitting this block mean that an error occurred and so nothing happened? Maybe you could log it/show error message. Similarly for others.